packages/backend/src/server/web/feed.ts

```diff
@@ -2,20 +2,22 @@ import { Feed } from 'feed';
 import { In, IsNull } from 'typeorm';
 import config from '@/config/index.js';
 import { User } from '@/models/entities/user.js';
-import { Notes, DriveFiles, UserProfiles, Users } from '@/models/index.js';
+import { Users, Notes, DriveFiles, UserProfiles } from '@/models/index.js';
+import { In, IsNull } from 'typeorm';
 
-export default async function(user: User) {
+export default async function(user: User, withAll = false, history = 5) {
 	const author = {
 		link: `${config.url}/@${user.username}`,
-		name: user.name || user.username,
+		email: `${user.username}@${config.host}`,
+		name: user.name || user.username
 	};
 
 	const profile = await UserProfiles.findOneByOrFail({ userId: user.id });
 
 	const notes = await Notes.find({
 		where: {
 			userId: user.id,
-			renoteId: IsNull(),
+			//renoteId: IsNull(),
```

Comment:
If you do not want to filter this, just remove the line instead of commenting it out.